test/nn/aggr/test_sort.py

```diff
@@ -1,78 +0,0 @@
-import torch
```

Comment:
Could you please clarify why the tests are removed?

Reply:
Mistake, added back.